tests/Unit/Contrib/JaegerSpanConverterTest.php

```diff
@@ -29,10 +30,10 @@ public function test_should_convert_an_otel_span_to_a_jaeger_thrift_span()
 
         [$convertedSpan] = (new SpanConverter())->convert([$span]);
 
-        $this->assertSame(0, $convertedSpan->traceIdLow);
-        $this->assertSame(0, $convertedSpan->traceIdHigh);
-        $this->assertSame(0, $convertedSpan->spanId);
-        $this->assertSame(0, $convertedSpan->parentSpanId);
+        $this->assertSame(PHP_INT_MIN, $convertedSpan->traceIdLow); //For some reason hardcoding -9223372036854775808 (-2^63) instead of PHP_INT_MIN isn't liked by PHPUnit. Might be a high precision arithmetic issue?
```

Comment:
I don't think the comments need to stay. Using PHP_INT_MIN is more readable and less mysterious anyway.

Reply:
Found a [bug](https://bugs.php.net/bug.php?id=76385) reported in PHP. Quoting one comment from the bug:

`This bug is hilarious:
The problem is that PHP parses "-9223372036854775808" not as a negative number but as a "-" followed by "9223372036854775808". The latter value is, of course, just barely too large for an integer so it overflows to a float.`

Reply:
Wow that's so interesting. Great find @kishannsangani !

And went ahead and removed the comments